packages/pluggableWidgets/popup-menu-native/src/__tests__/PopupMenu.spec.tsx

```diff
@@ -1,6 +1,6 @@
 import { PopupMenuProps } from "../../typings/PopupMenuProps";
 import { PopupMenuStyle } from "../ui/Styles";
-import { Text, TouchableOpacity } from "react-native";
+import { Text, TouchableHighlight } from "react-native";
```

Comment:
Looks suspicious 😛

Reply:
Yep, no idea how it was working before. But if you check implementation, you can see that the new test is more correct :)